op-challenger/game/fault/agent.go

```diff
@@ -112,6 +115,10 @@ func (a *Agent) shouldResolve(status gameTypes.GameStatus) bool {
 // tryResolve resolves the game if it is in a winning state
 // Returns true if the game is resolvable (regardless of whether it was actually resolved)
 func (a *Agent) tryResolve(ctx context.Context) bool {
+	if err := a.resolveClaims(ctx); err != nil {
+		a.log.Error("Failed to resolve claims", "err", err)
+		return false
+	}
```

Comment:
Good point. It only technically works because the game actor periodically checks if the game is resolvable. So `resolveClaims` also gets called periodically.
But it'll be better to do as you've described here rather than rely on the game scheduler to wake up the actor.